README.md

```diff
@@ -1,44 +1,84 @@
 # ENERGY OUTREACH COLORADO
-### KNOW THE TRUE LEVEL OF NEED IN CO
+### KNOW THE TRUE LEVEL OF ENERGY NEED IN CO
 
-The general idea of the project is to provide an understanding of how much need is in a specific place.
+The Energy Assistance Dashboard is an interactive single-page web
+application for visualizing and tracking data related to low-income
+energy assistance through [Energy Outreach Colorado](https://www.energyoutreach.org/).
 
+The purpose of the dashboard is to provide an understanding of how
+much energy need is in a specific place.
+
+## About
+### How It Works
+There are really two parts to this web application:
+
+1. The first part is the user-facing side, which is a web page that
+   displays a dashboard of energy visualizations. The design of this
+   dashboard will be based on the version 1 visualization mockup
+   found below in the [Google Sheet](#resources).
+
+2. The other part is the admin-facing side, which is an
+   [Airtable](https://airtable.com) document that stores the data
+   which feeds the user-facing visualizations. Currently, the data
+   can also be found below in the [Google Sheet](#resources).
+
+The idea is that administrators will be able to easily update
+the Airtable document and users will see changes reflected in the web
+dashboard.
+
+### Project Goals
 The must haves of the project are that it:
 
-1. Is printable. 
+1. Is printable.
 2. Easily used by non-technical users. This can be interpreted in two ways. One is just being able to provide a link to someone and then they have something they can reference and easily print out. The other way is whoever works
 
 Stretch goal ideas:
 * Automating data collection
 * Add more types of data
 
-V1 of the project
-https://docs.google.com/spreadsheets/u/3/d/1e6nSz5Mw8kloFNkSN-xi2IjnSP_pclgJy7U1C1fBx68/edit?usp=sharing
+### Resources
 
-## Backend Setup
+- [1]: Version 1 visualization mockup and data can be found in this
+  Google Sheet: [Colorado Low Income Energy Stats](https://docs.google.com/spreadsheets/u/3/d/1e6nSz5Mw8kloFNkSN-xi2IjnSP_pclgJy7U1C1fBx68/edit?usp=sharing)
+  - The first page, titled `County Lookup - Report View` is the
+    visualization dashboard that we will be re-building as a web page
+  - Other pages contain data that will feed the dashboard
+
+- [2]: Pitch Slides: https://docs.google.com/presentation/d/1_J7BpB2Spzu1cWV8QD1gIGR2H1vUTuCrK4WGlpoAk9A
+
+## Development
+On the technology side, the project is split into two parts, a backend
+HTTP API and a web frontend. The backend API is being built with
+Python 3.6+ and [FastAPI](https://fastapi.tiangolo.com/). The
+frontend stack is still being decided.
+
+### Backend Setup
 1. Install python3
 
-1. Navigate to the project folder and setup a virtual environment named `venv` by running the command:
+2. Navigate to the project folder and setup a virtual environment named `venv` by running the command:
```

Comment:
If you can keep these list numbers as `1` throughout, it would allow changing the order / inserting new items without having to renumber every item. Personal preference!

Reply:
@pjhoberman thanks, I didn't know there was a reason for doing that. Good to know, I will change that back.